.devcontainer/images/Dockerfile.base

```
@@ -0,0 +1,145 @@
+        openvpn wireguard-tools strongswan pptp-linux \
+        libwebkit2gtk-4.1-dev \
+        libxdo-dev \
+        libayatana-appindicator3-dev \
+        librsvg2-dev && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# =============================================================================
+# PPA-Based Tools (HashiCorp, 1Password, Ansible) - consolidated
+# =============================================================================
+RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)